test/e2e/mongodbtests/mongodbtests.go

```diff
@@ -340,6 +340,27 @@ func Connect(mdb *mdbv1.MongoDBCommunity, opts *options.ClientOptions) error {
 	})
 }
 
+// Connect performs a connectivity check by initializing a mongo client
```

Comment:
Can we make sure that the first word of the docstring is the correct name of the method?